tests/integration/test_events.py

```diff
@@ -1,241 +0,0 @@
-from __future__ import absolute_import
-
-from ddt import data, ddt, unpack
-from mock import Mock, patch
-from selenium.webdriver.common.keys import Keys
```

Comment:
@feanil is it safe and fine to remove selenium along with bok-choy, apparently it looks like bok-choy uses selenium and there is no use of selenium without bok-choy but is that the case?

Reply:
Yes, selenium and bok-choy should be removed together.